tests/unit/lms/services/jwt_test.py

```diff
@@ -111,20 +112,23 @@ def test_decode_lti_token_with_empty_token(self, svc):
     def test_decode_lti_token_with_no_kid(self, svc):
         jwt_str = self.encode_jwt({"key": "value"}, headers={"key": "value"})
 
-        assert not svc.decode_lti_token(jwt_str)
+        with pytest.raises(ValidationError):
+            svc.decode_lti_token(jwt_str)
```

Comment:
You might also want to test that it raises the right error message? This can catch issues with either the code or the test set up (when a test isn't testing the situation that it thinks it is, but is actually triggering a different `ValidationError`). It frequently catches a mistake.

```python
with pytest.raises(ValidationError) as exc_info:
    svc.decode_lti_token(jwt_str)

assert exc_info.err.message == { ... }
```